modules/virt-edit-cdrom-vm.adoc

```diff
@@ -19,27 +19,21 @@ endif::[]
 [id="virt-vm-edit-cdrom_{context}"]
 
 = Editing CD-ROMs for {object-gui}
-Use the following procedure to configure CD-ROMs for virtual machines.
+Use the following procedure to edit CD-ROMs for {object}s.
 
 .Procedure
 
-. From the *{object-gui}* tab, select your {object}.
+. Click *Workloads* -> *Virtualization* from the side menu.
 
-. Select the *Overview* tab.
+. Click the *{object-gui}* tab.
 
-. To add or edit a CD-ROM configuration, click the pencil icon to the right of the *CD-ROMs* label. The *Edit CD-ROM* window opens.
+. Click the *Disks* tab.
 
-* If CD-ROMs are unavailable for editing, the following message displays: *The virtual machine doesn't have any CD-ROMs attached.*
+. To edit a CD-ROM configuration, click the Options {kebab} menu for the CD-ROM and select *Edit CD-ROM.*
```

Comment:
```suggestion
. To edit a CD-ROM configuration, click the Options {kebab} menu for the CD-ROM and select *Edit*
```

Reply:
@yaacov Edit was made. Appreciate the quick turnaround for your review :-)